std_msgs/Header header
uint8 camera_index
uint16 image_width
uint16 image_height
uint8[] image
